src/bindings/python/flux/job/kvslookup.py

```diff
@@ -53,16 +53,11 @@ def job_info_lookup(flux_handle, jobid, keys=["jobspec"]):
     return rpc
 
 
-def _original_setup(keys, original):
```

Comment:
commit message: is "inputted" a word?

Reply:
https://english.stackexchange.com/questions/30684/inputted-or-input

Reply:
LOL

https://www.merriam-webster.com/grammar/is-inputted-a-word

> the issue of how to treat the past participle is a contentious one, with much blood being shed on both sides.

But it seems folks have accepted as a word

:shrug:

Reply:
:+1: However from now on I will think of it as a golf term.